pkg/command/plugin.go

```diff
@@ -311,6 +292,41 @@ func newInstallPluginCmd() *cobra.Command { //nolint:funlen
 	return installCmd
 }
 
+func installPluginsForPluginGroup(cmd *cobra.Command, args []string) error {
+	var pluginName string
+	// We are installing from a group
+	if len(args) == 0 {
+		// Default to 'all' when installing from a group
+		pluginName = cli.AllPlugins
+	} else {
+		pluginName = args[0]
+	}
+
+	if pluginName == cli.AllPlugins {
+		pg, groupIdentifier, err := pluginmanager.GetPluginsFromGroup(group)
+		if err != nil {
+			return err
+		}
+		groupIdentifier.Version = pg.RecommendedVersion
+		groupIDAndVersion := fmt.Sprintf("%s-%s/%s:%s", groupIdentifier.Vendor, groupIdentifier.Publisher, groupIdentifier.Name, groupIdentifier.Version)
+		log.Infof("The following plugins will be installed from plugin group '%s'", groupIDAndVersion)
+		displayGroupContentAsTable(pg, groupIdentifier.Version, "", false, false, cmd.OutOrStdout())
```

Comment:
If you run `tz plugin group install --group vmware-tkg/default > out`, you will notice that only the new table printout goes to the file, the rest of the output goes to `stderr`.  I feel the new table should also go to `stderr` along with the rest.

You can do this by passing `cmd.ErrOrStderr()` above to `displayGroupContentAsTable()` in this scenario.

**Explanation**:
When the user runs the command `tz plugin install --group` they are not asking for printed output, but for an action to be taken (compared to `tz plugin list` for example, where the user is asking for output).

This is why the `tz plugin install` command prints _logs_ which go to `stderr` while `tz plugin list` prints _output_ which goes to `stdout`.

Reply:
Yes got it, fixed.